### ISSUE TITLE
[Partial Fix Included] Lvl 28 PvP Gear No Rep Requirements

### ISSUE THREAD

**Issue by StraysFromPath** [ZhengPeiRu21/mod-individual-progression#226]
The lvl 28 defilers and highlander gear has no rep requirement atm. This code changes it to need friendly, since if you do enough BGs to reach revered you will be a minimum of lvl 45 from BG experience.

Friendly at 28 is achievable and relevant to the intent and better than current which is no requirement at all.

```
UPDATE `item_template` SET `RequiredReputationFaction` = 510, `RequiredReputationRank` = 4 WHERE (`entry` = 20152);
UPDATE `item_template` SET `RequiredReputationFaction` = 510, `RequiredReputationRank` = 4 WHERE (`entry` = 20157);
UPDATE `item_template` SET `RequiredReputationFaction` = 510, `RequiredReputationRank` = 4 WHERE (`entry` = 20162);
UPDATE `item_template` SET `RequiredReputationFaction` = 510, `RequiredReputationRank` = 4 WHERE (`entry` = 20164);
UPDATE `item_template` SET `RequiredReputationFaction` = 510, `RequiredReputationRank` = 4 WHERE (`entry` = 20169);
UPDATE `item_template` SET `RequiredReputationFaction` = 510, `RequiredReputationRank` = 4 WHERE (`entry` = 20172);
UPDATE `item_template` SET `RequiredReputationFaction` = 510, `RequiredReputationRank` = 4 WHERE (`entry` = 20178);
UPDATE `item_template` SET `RequiredReputationFaction` = 510, `RequiredReputationRank` = 4 WHERE (`entry` = 20182);
UPDATE `item_template` SET `RequiredReputationFaction` = 510, `RequiredReputationRank` = 4 WHERE (`entry` = 20188);
UPDATE `item_template` SET `RequiredReputationFaction` = 510, `RequiredReputationRank` = 4 WHERE (`entry` = 20191);
UPDATE `item_template` SET `RequiredReputationFaction` = 510, `RequiredReputationRank` = 4 WHERE (`entry` = 20197);

UPDATE `item_template` SET `RequiredReputationFaction` = 509, `RequiredReputationRank` = 4 WHERE (`entry` = 20090);
UPDATE `item_template` SET `RequiredReputationFaction` = 509, `RequiredReputationRank` = 4 WHERE (`entry` = 20093);
UPDATE `item_template` SET `RequiredReputationFaction` = 509, `RequiredReputationRank` = 4 WHERE (`entry` = 20096);
UPDATE `item_template` SET `RequiredReputationFaction` = 509, `RequiredReputationRank` = 4 WHERE (`entry` = 20099);
UPDATE `item_template` SET `RequiredReputationFaction` = 509, `RequiredReputationRank` = 4 WHERE (`entry` = 20102);
UPDATE `item_template` SET `RequiredReputationFaction` = 509, `RequiredReputationRank` = 4 WHERE (`entry` = 20105);
UPDATE `item_template` SET `RequiredReputationFaction` = 509, `RequiredReputationRank` = 4 WHERE (`entry` = 20108);
UPDATE `item_template` SET `RequiredReputationFaction` = 509, `RequiredReputationRank` = 4 WHERE (`entry` = 20111);
UPDATE `item_template` SET `RequiredReputationFaction` = 509, `RequiredReputationRank` = 4 WHERE (`entry` = 20114);

```

**Comment by StraysFromPath** [comment]
If you want blizzlike change the rep rank = 4 to 6, but be aware this makes the boots completely unuseable.

The amount of BGs needed to get from neutral to revered would take you from 20 to 45 at which point lvl 28 boots are useless.

**Comment by ZhengPeiRu21** [comment]
Thanks very much for submitting this fix! I did use the original Vanilla rank of 6 rather than 4, even though that means the gear most likely won't be viable once it's available - I'd like to avoid the slippery slope of trying to fix Vanilla itemization.

**Comment by StraysFromPath** [comment]
> Thanks very much for submitting this fix! I've committed your fix in [77a65bd](https://github.com/ZhengPeiRu21/mod-individual-progression/commit/77a65bd50602f5e387c65c45e575889bb9211260). I did use the original Vanilla rank of 6 rather than 4, even though that means the gear most likely won't be viable once it's available - I'd like to avoid the slippery slope of trying to fix Vanilla itemization.

Yes and no.

In wotlk all pvp faction reps are harder to get. The only way to get WSG rep is 35 rep for capping a flag so maximum 105 rep per game)

In Vanilla you got battleground marks for bgs which were used for quests like this:

https://classicdb.ch/?quest=7789

Receiving 500 rep for 1 WSG victory which could be completed at  10/20/30/40/50/60.

That's 6 x 500 = 3000 rep. Which gets you well into friendly.

+ +75 rep for winning wsg - https://classicdb.ch/?quest=8296

+ +50 rep for losing WSG - https://classicdb.ch/?quest=8266

+ +35 per flag capped (like wotlk)

+ +6x100 = 600 for these quests https://classicdb.ch/?quest=8368

Basically you actually could get revered in WSG at lvl 18 in vanilla quite easily - especially since there was no BG xp in vanilla.

This is impossible in wotlk/3.3.5.

---

If you played 50 BGs, won 25 with 2 caps, lost 25 with 1 cap -

in wotlk you would have 5250 rep (friendly) from caps

in vanilla you would more than double the rep as you have 5250 from caps, 3600 from quests, 1875 (75x25) from wins, and 625 (25x25) from losses totalling 11350 rep (honoured)

It would be more accurate to vanilla to change all 18 and 28 gear to friendly, 38 and 48 gear to honoured, and 58 gear to revered.
